Orange/widgets/visualize/owdistributions.py

```diff
@@ -209,7 +209,7 @@ def __init__(self):
         self.groupvarview = gui.comboBox(
             box, self, "groupvar_idx",
             callback=self._on_groupvar_idx_changed,
-            valueType=str, contentsLength=12)
```

Comment:
This `valueType` is wrong, but `gui.comboBox` ignored it anyway since `sendSelectedValue` was left `False`.